stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/composer/MessageComposerState.kt

```diff
@@ -25,4 +28,6 @@ public data class MessageComposerState(
     val mentionSuggestions: List<User> = emptyList(),
     val commandSuggestions: List<Command> = emptyList(),
     val cooldownTimer: Int = 0,
+    val messageMode: MessageMode = MessageMode.Normal,
+    val showInChannel: Boolean = false,
```

Comment:
This is a bit vague as to what is `showInChannel` - what will we show? The message, the composer etc.

Maybe renaming to `alsoSendToChannel` might be more clear.

But I'm not 100% sure as it's the same component for 2 places where we have messages.

Reply:
or the other variation could be -> `sendsMessageToChannel` :)

Reply:
Yeah, that's true `showInChannel` is indeed misleading here. Will use a more verbose alternative here.